source/connect/tls.txt

```diff
@@ -0,0 +1,245 @@
+- Specify the client TLS certificate.
+- Specify the CA certificate to verify the server's TLS certificate.
+
+To learn how to configure your MongoDB deployment for TLS, see the
+:manual:`TLS configuration guide </tutorial/configure-ssl/>` in the
+{+mdb-server+} manual.
+
+.. note::
+
+   This page assumes prior knowledge of TLS/SSL and access to valid certificates.
+   A full description of TLS/SSL, PKI (Public Key Infrastructure) certificates, and
+   Certificate Authorities (CAs) is beyond the scope of this documentation. To
+   learn more about TLS, see the Wikipedia entry for :wikipedia:`Transport Layer Security <Transport_Layer_Security>`.
+
+.. _ruby-enable-tls:
+
+Enable TLS
+----------
+
+You can enable TLS for the connection to your MongoDB deployment in the following ways:
+
+- Set the ``ssl`` option to ``true`` in a new ``Mongo:Client`` object
+- Set the ``tls`` option to ``true`` in your connection string
+
+.. note:: SSL Naming Convention
+
+    All {+mdb-server+} versions supported by the Ruby driver (2.6 and higher) 
+    only implement TLS. 2.6 and do not use SSL.
+    
+    For historical reasons, the Ruby option names pertaining to TLS configuration 
+    use the ``ssl`` rather than the ``tls`` prefix. The next major version of the Ruby 
+    driver (3.0) will use the ``tls`` prefix for Ruby option names.
+
+.. tabs::
+
+    .. tab:: Mongo::Client
+        :tabid: mongoclient
+
+        To configure certificates, you must specify the following options:
+
+        - ``ssl_cert``: The certificate file used to identify the connection against MongoDB.
+        - ``ssl_key``: The private keyfile used to identify the connection against MongoDB.
+        - ``ssl_ca_cert``: The file containing the concatenated CA certificates 
+          used to validate certs passed from the other end of the connection.
+          By default, the driver will use the default system root certificate 
+          store as the trust anchor, if the option is not designated. 
+
+        These certificate options can be specified as different types. For example, the 
+        ``ssl_cert`` option has the alternative ``ssl_cert_object`` and ``ssl_cert_string`` 
+        options, to specify the certificate file as either a certificate object or a string.
+        For more information on certificate options, see the Options Hash section of 
+        the API Documentation for `Mongo::Client <{+api-root+}/Mongo/Client.html>`__ .  
+
+        In the following example, the TLS certificate and corresponding private key 
+        are provided in separate files:
+
+        .. literalinclude:: /includes/connect/tls.rb
+            :language: ruby
+            :dedent:
+            :start-after: start-enable-tls-settings
+            :end-before: end-enable-tls-settings
+        
+        The TLS certificate and private key can be contained in the same file, but 
+        both the certificate and private key options must still be specified. In the 
```

Comment:
s: shortened

```suggestion
        You can specify both the TLS certificate and private key in a single file.
        In the 
```

Reply:
Thanks for the edit! Will keep in the part about still needing to specify both options, as I think it's important to clarify.